shallowblue_rllib.py

```diff
@@ -113,15 +126,15 @@ def step(self, action):
             self.agent_host.sendCommand('turn 0')
             self.agent_host.sendCommand('jump 0')
             self.agent_host.sendCommand('attack 1')
-            time.sleep(1.5)
-        elif (self.air_level < 0.3 * self.full_air) and action[2] > 0:
+            time.sleep(2)
+        elif (self.air_level < self.air_threshold * FULL_AIR) and action[2] > 0:
             self.agent_host.sendCommand('move 0')
             self.agent_host.sendCommand('turn 0')
             self.agent_host.sendCommand('attack 0')
             self.agent_host.sendCommand('jump 1')
-            time.sleep(0.2)
+            time.sleep(1)
```

Comment:
increased sleep to try and make sure no other actions stop it from falling when its already initiated a jump 1